docs/feature-flags.md

```diff
@@ -0,0 +1,21 @@
+# Feature flags
+
+Feature flags can be used to programatically turn features on and off. We can use them to control the release of features in different environments or at different times.
+
+## Adding a flag
+
+To create a flag you need to add it as a constant to `DfE.FindInformationAcadmiesTrusts/Configuration/FeatureFlags.cs` and add a default value to the appsettings.json file. Finally add the flag name, default value (used in appsettings.json) and reason for creating the flag/what the flag controls to the list at the end of this file.
+
+## Using a flag
```

Comment:
A link to the Microsoft docs would be useful here for further reading and also to show that we're using the standard Microsoft way of doing things